core/arch/arm/kernel/secure_partition.c

```diff
@@ -120,7 +120,9 @@ TEE_Result sp_partition_info_get(uint32_t ffa_vers, void *buf, size_t buf_size,
 {
 	TEE_Result res = TEE_SUCCESS;
 	uint32_t part_props = FFA_PART_PROP_DIRECT_REQ_RECV |
-			      FFA_PART_PROP_DIRECT_REQ_SEND;
+			      FFA_PART_PROP_DIRECT_REQ_SEND |
```

Comment:
Aren't these properties owned by the SP? Shouldn't these be read from the manifest?

Reply:
Done